content/blog/windows-fsx-joined-to-active-directory-using-aws-cdk.md

```diff
@@ -0,0 +1,280 @@
+---
+id: blog
+title: Deploying Windows FSx joined to an AD using AWS CDK
+description: This blog describes how to join a Windows FSx file server to an AWS managed Active Directory using AWS CDK.
+author: Nidhi Sharma
+tags:
+  - AWS CDK
+  - Windows FSx
+  - Active Directory
+slug: windows-fsx-joined-to-ad-with-aws-cdk
+date: 2023-02-25
+---
+
+In a Windows Server setup, it is common practice to use Active Directory for server and user management. For this purpose, you can create an AWS managed Active Directory. Any EC2 instances or other resources that you would like to manage should join the AD upon their creation. In our setup, we required a common file storage for sharing resources across our Windows EC2 instances. For this purpose, we used AWS FSx for Windows.
```

Comment:
Suggestion: "In our setup, we required...." should probably be rewritten to "At a client, we had a setup that required...".